tests/unit/conftest.py

```diff
@@ -149,13 +149,12 @@ def mock_airflow_connection_datarobot_basic_credentials(mocker, dr_basic_credent
     mocker.patch.dict("os.environ", AIRFLOW_CONN_DATAROBOT_BASIC_CREDENTIALS_TEST=conn.get_uri())
 
 
-@pytest.fixture(autouse=True)
+@pytest.fixture()
 def mock_datarobot_gcp_credentials(mocker, dr_gcp_credentials_conn_details):
     gcp_credentials_create_mock = mocker.Mock(
         credential_id='test-gcp-credentials-id',
         name='datarobot_gcp_credentials_test',
         credential_type='gcp',
-        gcp_key=dr_gcp_credentials_conn_details["gcp_key"],
```

Comment:
ditto, why is the secret being deleted?

Reply:
it was previously added by mistake, its mock for Credential object which doesn't contain anything except credential_id, name, credential_type and description